src/Tizen.Network.Nfc/Tizen.Network.Nfc/NfcNdefRecord.cs

```diff
@@ -284,6 +284,11 @@ public NfcNdefRecord(string mimeType, byte[] data, uint dataSize)
             }
         }
 
+        internal NfcNdefRecord(IntPtr recordHandle)
+        {
+            _recordHandle = recordHandle;
```

Comment:
You need to control ownership of native handle. 
You do better to use `SafeHandle` instead of IntPtr 
https://docs.microsoft.com/ko-kr/dotnet/api/system.runtime.interopservices.safehandle?view=netframework-4.8